digital-twin/Samples/device/AndroidSample/app/build.gradle

```diff
@@ -0,0 +1,41 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileOptions {
+        sourceCompatibility 1.8
+        targetCompatibility 1.8
+    }
+    compileSdkVersion 29
+    buildToolsVersion "29.0.2"
+    defaultConfig {
+        applicationId "com.microsoft.azure.sdk.iot.digitaltwin.sample"
+        minSdkVersion 19
+        targetSdkVersion 29
+        versionCode 1
```

Comment:
Can we specify a wild card for the latest android API as oppose to hard-coding it with 29 as target?

Reply:
Not a good idea.